docs.go

```diff
@@ -0,0 +1,18 @@
+/*
+#  #############################################
```

Comment:
we need to correct the copyright and include the apache 2 license

Reply:
```
/*
Copyright 2019 The Beehive Authors.

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/

```

is it ok ?

Reply:
I think we can use KubeEdge Authors as beehive is a subproject under KubeEdge organization

Reply:
@sids-b ok.  but  do all the golang source files need to be added this license as well as kubeedge project?